src/components/PrimaryNav/PrimaryNav.js

```diff
@@ -168,10 +181,23 @@ const PrimaryNav = () => {
               target="_blank"
               rel="noopener noreferrer"
             >
-              <img className="p-list__icon" src={bugIcon} alt={`bug icon`} />
               Report a bug
             </a>
           </li>
+          <li className="p-list__item">
+            <a
+              rel="noopener noreferrer"
+              target="_blank"
+              className="p-list__link"
+              href="#_"
+              onClick={e => {
+                e.preventDefault();
+                window.usabilla_live("click");
```

Comment:
Is there a way to check if `window.usabilla_live` exists before displaying the link? The function was blocked by ad-blocker.

Reply:
This is an interesting problem as I wonder the percentage of users that will block this link but will also want to provide feedback. Maybe usabilla has a link with query params that we can use instead of their script?

Reply:
investigated this and couldn't find a timely resolution. 

@anthonydillon If you want to investigate further, can you raise a ticket?

Reply:
I created the issue as we should definitely address this as it's just a broken link for many. 

https://github.com/canonical-web-and-design/jaas-dashboard/issues/383